content/W2015/W15-event-boardgames.md

```diff
@@ -19,3 +19,6 @@ Hope to see you there!
 + **What:** Board Games Meet and Greet
 + **Where:** Math CnD (MC 3002)
 + **When:** Thu. Jan. 8, 7:00&ndash;11:00PM
+
+# About Big CSters #
+Big CSters is our mentorship program for female Computer Science students. We host social activities throughout the term for CSters to meet new friends in all stages of university life.
```

Comment:
Hm, I would change this description. I don't know if it encompasses everything we want in our CSters program (mentoring being the big one); also, @annalorimer and I had discussed using more inclusive language, since we're not just limiting to cis-women in CS. Maybe you should leave this commit for later.